### ISSUE TITLE
Consider remodelling of "Format (by type of data" concepts

### ISSUE THREAD

**Issue by joncison** [edamontology/edamontology#287]
There are two problems:

- children of  _"Format (by type of data)"_ (http://edamontology.org/format_2350 ) do not always exactly match concepts in the _Data_ (http://edamontology.org/data_0006 )branch
- children of these children are currently explicitly defined via SubClass relations

leading to:
- inconsistency / confusion between format and data concepts
- lack of sustainability (it's an extra overhead to explicitly define SubClass relations)

A solution would be to replace the immediately ancestors of _"Format (by type of data)"_ with a bunch of defined (equivalent) OWL classes, i.e. we define the OWL logic of what, say a "Sequence format" is and then infer the ancestors automatically.

I'd need to look into exactly how to do this, but I believe it's possible using OWL syntax.

cc @matuskalas FHI

**Comment by veitveit** [comment]
Is there any more concrete plan to automatically connect _format (by type of data) concepts_ with _data_? 

Nailing this one would make the formats much more consistent and easier to add/edit.

Automatic remodeling of e.g. the _is_format_of_ into into _format concepts_ would require to deal with the following challenges:
-  Starting point for creating automatic relations:  _is_format_of_ into or _format concepts_?
:construction: Possible solution: Make _is_format_of_ the defining item. This would also elevate this relation into something that can easily be extracted. Converting ancestors into _is_format_of_ should be doable without too much effort.
- Multiple _data_ per format: Does that imply multiple parents?
:construction: Possible solution: Not sure here. How bad would it be to have multiple (even >10) parents for a format? This could still work but we need to set a limit and also think a way to deal with the generic formats like _FASTA_ and _Textual format_
- Inheritance of related data concepts to children: What if a child is more specialized and does not include all data concepts?
:construction: Possible solution: Maintain inheritance as is but take the _is_format_of_ as the defining term. 

I consider the parent relations of the formats a very useful feature to find appropriate formats when browsing through the EDAM tree. Are there more applications or could the rest be taken over by the __is_format_of__?

@joncison @hmenager @matuskalas

**Comment by joncison** [comment]
I'm not sure (need to think more).  To explain a bit my first post ... in OWL you can define something like _e.g._

"Sequence alignment format"  is-equivalent-to   {some **Format** _isFormatOf_ "Sequence alignment"}

and then run a reasoner over EDAM.owl - every Format concept where {_isFormatOf_ "Sequence alignment"} is defined would be inferred to be a child of "Sequence alignment format".  This means you wouldn't need anymore to manually maintain relationships of the type e.g. {"Clustal" _subClassOf_ "Sequence alignment format"}.  But we would need to save (and eventually distribute) the inferred version of EDAM - but keep the non-inferred version for maintenance (i.e. two EDAM files ... this is what we did at EBI for EFO ontology).

In any case, the "placeholder" concepts (which could be defined by equivalence statements, as above) should exactly correspond to concepts in **Data** subontology. I don't think they do currently!

Sorry, this doesn't really pick up on your points above, but I need to think more :)

**Comment by veitveit** [comment]
Yes, there are quite a few placeholders that do not directly correspond to _data_ concepts. But they are easy to map as in most cases there is only a slight change in the wording
